R/options.R

```diff
@@ -16,13 +16,22 @@ get_plot_options <- function() {
     return(as.list(plot.opts))
 }
 
-# all plot mime types. default value for the jupyter.plot_mimetypes option
-plot_mimetypes <- c(
-    'image/png',
-    'application/pdf',
-    'image/svg+xml')
+opt.defaults = list(
+    jupyter.rich_display = TRUE,
+    jupyter.result_mimetypes = c(
+        'text/plain',
+        'text/html',
+        'text/markdown',
+        'text/latex'),
```

Comment:
[and many more](http://ipython.org/ipython-doc/stable/api/generated/IPython.core.formatters.html#IPython.core.formatters.format_display_data). i’ll add them.